exporter/exporterhelper/tracehelper_test.go

```diff
@@ -161,27 +152,24 @@ func checkRecordedMetricsForTraceExporter(t *testing.T, te exporter.TraceExporte
 	ctx := observability.ContextWithReceiverName(context.Background(), fakeReceiverName)
 	const numBatches = 7
 	for i := 0; i < numBatches; i++ {
-		if err := te.ConsumeTraceData(ctx, td); err != wantError {
-			t.Fatalf("Want %v Got %v", wantError, err)
-		}
+		err := te.ConsumeTraceData(ctx, td)
```

Comment:
nit: you can make this one line and have the method invoked in the require.Equal